### PR TITLE
created the start screen and added the Spanish game option. now we ne…

### PR DESCRIPTION
**Pull request by pranavramesh**
…ed to connect the Spanish game button to the Spanish instance that needs to be created. Let me know when the Spanish class has been completed and I can work on implementing that with the front end...

### CODE REVIEW

File: src/main/java/assets/css/custom.css [hunk 1]
```diff
@@ -1,3 +1,2 @@
 /* Add additional stylesheets below
--------------------------------------------------- */
-
+-------------------------------------------------- */
```
Comment (mihaidan) [inline src/main/java/assets/css/custom.css:2]: What's the change that was made here? @pranavramesh

Comment (pranavramesh) [inline src/main/java/assets/css/custom.css:2]: There was actually no change made there. I accidentally modified this file and then deleted the contents, hence why it shows as if there was actually something of value. I ended up using AcesUpFinal.css instead. @mihaidan

**Comment by pranavramesh** [comment]
@hardergeorge As far as I know, this is what will make the call to the controller. I will do some more research on how we can accomplish that to see if this is the best way.

**Comment by hardergeorge** [comment]
@pranavramesh cool I was mostly just curious

**Pull request merged**